cishouseholds/pipeline/run.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import yaml
+
+from cishouseholds.pipeline.bloods_delta_ETL import bloods_delta_ETL  # noqa F401
+from cishouseholds.pipeline.sample_delta_ETL import sample_delta_ETL  # noqa F401
+from cishouseholds.pipeline.survey_responses_version_2_ETL import survey_responses_version_2_ETL  # noqa F401
+from cishouseholds.pipeline.test_ETL import test_ETL  # noqa F401
+
+
+def run_from_config(config_location: str):
+    with open(config_location) as fh:
+        read_data = yaml.load(fh, Loader=yaml.FullLoader)
+    print("Data: ", read_data)
+    if type(read_data) != list:
+        read_data = [read_data]
+    for ETL in read_data:
+        if ETL["run"]:
+            for loc in globals():
+                print("local: ", loc)
+            output_df = globals()[ETL["function"]](ETL["resource_path"])
```

Comment:
I really like the config format here, especially how we can switch sections on or off.

Referencing objects from `globals` is generally considered as bad practice. One elegant way of dealing with this might be to create a `@pipeline_section` decorator, which adds the high level pipeline function to a dictionary of pipeline sections. There are a couple of different approaches in [this example](https://stackoverflow.com/a/54075852/8103477), depending on whether you want to use the function's exact name or an alias.